src/store/actions/actions-model.ts

```diff
@@ -262,10 +266,6 @@ export const setValidFileNumber: ActionIF = ({ commit }, isValid: boolean): void
   commit('mutateValidFileNumber', isValid)
 }
 
-export const setValidResolutionDate: ActionIF = ({ commit }, isValid: boolean): void => {
```

Comment:
Was not used so I deleted it.

Reply:
There must have been some refactoring that handles that validation differently?

Reply:
I just looked at the code and don't understand how this is working... but it is.

I will come back to this comment. Please leave it unresolved.

Reply:
Found it.

The ResolutionDates component itself sets the Valid Component key (which App checks to see if there are any invalid sections and whether to display the Actions message).

It looks like `setValidResolutionDate` was left over from a previous implementation, which was replaced by the Valid Component k-v object.